src/Okta.Sdk.IntegrationTests/EventHooksClientShould.cs

```diff
@@ -0,0 +1,382 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    public class EventHooksClientShould
+    {
+        private const string SdkPrefix = "dotnet_sdk";
+        private const string EventType = "EVENT_TYPE";
+
+        private static readonly string[] TestEventItems = new string[]
+        {
+            "user.lifecycle.create",
+            "user.lifecycle.activate",
+        };
+
+        private static readonly string[] UpdatedTestEventItems = new string[]
+        {
+            "user.lifecycle.create",
+            "user.lifecycle.activate",
+            "user.lifecycle.deactivate",
+        };
+
+        private static readonly EventHookChannel TestEventHookChannel = new EventHookChannel
+        {
+            Type = "HTTP",
+            Version = "1.0.0",
+            Config = new EventHookChannelConfig
+            {
+                Uri = "https://www.example.com/eventHooks",
+                Headers = new List<IEventHookChannelConfigHeader>
+                {
+                    new EventHookChannelConfigHeader
+                    {
+                        Key = "X-Test-Header",
+                        Value = "Test header value",
+                    },
+                },
+                AuthScheme = new EventHookChannelConfigAuthScheme
+                {
+                    Type = "HEADER",
+                    Key = "Authorization",
+                    Value = "Test-Api-Key",
+                },
+            },
+        };
+
+        private static readonly EventHookChannel UpdatedTestEventHookChannel = new EventHookChannel
```

Comment:
Suggestion: I know we are trying to avoid duplicated code here. However, I think for testing in general, having inline code helps with readability, even if it means duplicated code. I think we should favor code readability here, especially because usually when we add a new API, the reviewer has to take a look at a considerable number of tests, and we should facilitate that process. Let me know your thoughts, I'm open to discussion.

Reply:
You make a good point; moving these definitions closer to where they are used will likely make the tests more easy to read and understand.  Since these definitions aren't referenced in a whole bunch of places I'll move them into the tests that rely on them.